examples/example_plugin/example_plugin/views.py

```diff
@@ -46,12 +47,12 @@ class DeviceDetailPluginTabTwoView(views.ObjectView):
     template_name = "example_plugin/tab_device_detail_2.html"
 
 
-class ExamplePluginHomeView(View):
+class ExamplePluginHomeView(LoginRequiredMixin, View):
```

Comment:
Would it make sense to just create a generic nautobot view that inherits from django.views.generic.View and LoginRequiredMixin?

Reply:
There are a lot of instances of those combinations in this PR alone, I think it makes sense. Should be non-blocking tho.

Reply:
I like the idea, will do.